torchaudio/functional.py

```diff
@@ -844,6 +844,162 @@ def compute_deltas(specgram, win_length=5, mode="replicate"):
 
 
 @torch.jit.script
+def gain(waveform, gain_db=1.0):
+    # type: (Tensor, float) -> Tensor
+    r"""Apply amplification or attenuation to the whole waveform.
+
+    Args:
+       waveform (torch.Tensor): Tensor of audio of dimension (channel, time)
+       gain_db (float) Gain adjustment in decibels (dB) (Default: `1.0`).
+
+    Returns:
+       torch.Tensor: the whole waveform amplified by gain_db
+    """
+    if (gain_db == 0):
+        return waveform
+
+    ratio = 10 ** (gain_db / 20)
+
+    return waveform * ratio
+
+
+@torch.jit.script
```

Comment:
By the way, we just changed how we are scripting, see #314. Fortunately, it's just cosmetic, since we've already made your code jitable :)
* We are no longer decorating with `@torch.jit.script`
* We are adding tests instead, see [here](https://github.com/pytorch/audio/blob/master/test/test_functional.py#L511) for instance.

Reply:
If you remove the `@torch.jit.script` from the PR, please add a corresponding test in `test/test_functional.py`.